release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1639,14 +1639,14 @@ In the table below, features are marked with the following statuses:
 |GA
 
 |CPU manager
-|GA
-|GA
+|-
```

Comment:
CPU manager is listed as GA in the 4.10 RNs, at least back until 4.8. Should this entry be completely removed now instead?